end-to-end/.jazzerjsrc

```diff
@@ -0,0 +1,4 @@
+{
+	"includes": ["target"],
```

Comment:
Project files should be included by default. I think `.jazzerjsrc` could be removed in this example.